api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java

```diff
@@ -120,7 +121,7 @@ public Workspace getByName(String ns, String name) {
   @Override
   public Workspace getRequired(String ns, String firecloudName) {
     Workspace workspace = get(ns, firecloudName);
-    if (workspace == null) {
+    if (workspace == null || (workspace.getWorkspaceActiveStatusEnum() == WorkspaceActiveStatus.DELETED)) {
```

Comment:
it makes sense to me that the get endpoint should not fetch deleted workspaces; would like to hear some other opinions.